doc/LEGAL.md

```diff
@@ -0,0 +1,7 @@
+Documentation (c) by ProjectMeshnet.
```

Comment:
Is this a legal entity? Do you have CLA signatures from the actual contributors of the documentation? If no to either then this statement is technically wrong. However you can state that all documentation is CC-BY-4.0 and it should be fair to assume that anyone contributing documentation has agreed to license /their/ copyright under this license, as I do with the GPLv3+ and the codebase.